executor/api/rest/middlewares.go

```diff
@@ -16,6 +17,13 @@ const (
 
 	contentTypeOptsHeader = "X-Content-Type-Options"
 	contentTypeOptsValue  = "nosniff"
+
+	corsAllowOriginHeader  = "Access-Control-Allow-Origin"
+	corsAllowOriginValue   = "*"
```

Comment:
The default values for these headers are very lax from the security perspective. Maybe these values should be set from a custom config and the defaults should be made more strict.

Reply:
👍 - I agree.